tests/monotouch-test/ObjCRuntime/RegistrarTest.cs

```diff
@@ -1286,7 +1286,7 @@ public void TestConstrainedGenericType ()
 						void_objc_msgSend_out_IntPtr (obj.Handle, Selector.GetHandle ("m2:"), out value);
 						Assert.AreEqual (IntPtr.Zero, value);
 
-						value = new IntPtr (0xdeadbeef);
+						value = new IntPtr ((unchecked ((int) 0xdeadbeef)));
```

Comment:
does it make it an `un` `deadbeef` ?

Reply:
Only 32% dead as opposed to 64% dead now.

Reply:
moar 🧟